### ISSUE TITLE
API to change the in-command colours?

### ISSUE THREAD

**Issue by IngwiePhoenix** [elves/elvish#1813]
### What new feature should Elvish have?

Right now, these are the default colors in Elvish:

![image](https://github.com/elves/elvish/assets/2893716/315281a4-5808-487e-b748-8aa33d74c142)

But I quite like the blue color scheme that Fish uses; so I would like to be able to set those colors myself, if possible.

I looked into the store and edit hooks; but couldn't find the related hooks.

### Output of "elvish -version"

0.20.1+official

### Code of Conduct

- [X] I agree to follow Elvish's [Code of Conduct](https://src.elv.sh/CODE_OF_CONDUCT.md).

**Comment by krader1961** [comment]
> And so far no one has been motivated enough to write the necessary code.

An example of being motivated to work on a change that might not otherwise happen any time soon is my work to replace the Elvish daemon and dependency on the BoltDB project with a far simpler, traditional, flat-file store for shell history. When I started using Elvish I often encountered errors involving the daemon. I haven't encountered any of those problems in several years but still dislike that Elvish interactive history is stored in a huge binary blob that requires a dependency of a large project and the need for all Elvish instances to talk to a daemon over RPC to manipulate the history store. So for approximately four months I have been working on a series of changes to replace that daemon and BoltDB dependency with a far simpler mechanism that stores the history in a simple JSON encoded flat-file.

I haven't felt strongly enough about being able to customize the editor theme because the existing default theme works well enough for me given I use a terminal with a "light" (rather than "dark") background. The Elvish default colors are different from those I customized Fish, Ksh, or Bash to use but are "good enough". Passionate users like yourself, @IngwiePhoenix, are encouraged to submit changes that will benefit all users.
